dev/NavigationView/NavigationView.xaml

```diff
@@ -158,6 +164,19 @@
                                     </VisualState.Setters>
                                 </VisualState>
                             </VisualStateGroup>
+
+                            <VisualStateGroup x:Name="PaneSeperatorStates">
+                                <VisualState x:Name="SeparatorVisible">
+                                    <VisualState.Setters>
+                                        <Setter Target="VisualItemsSeparator.Visibility" Value="Visible"/>
```

Comment:
The separatorVisible should be the default state, so should not have additional setters.

Reply:
Default state is SeparatorCollapsed, not the visible state. Fixed now and removed the setter for that.